### ISSUE TITLE
Unable to set fullscreen video mode

### ISSUE THREAD

**Issue by neuromancer** [Raptor007/X-Wing-Revival#1]
After compiling this game in ArchLinux and download the assets, it fails to start:

```
$  ./xwingrev 
X-Wing Revival 0.1.6 Alpha
Unable to set fullscreen video mode 0x0x32: Couldn't find matching GLX visual
Unable to set fullscreen video mode 640x480x32: Couldn't find matching GLX visual
Unable to set windowed video mode 640x480x32: Couldn't find matching GLX visual
```

I'm using a GeForce GTX 750 with the latest nvidia drivers (396.24)

**Comment by neuromancer** [comment]
It works disabling that line, but all the ships are completely black.

**Comment by Raptor007** [comment]
Hmm.  Try pressing tilde to bring up the in-game console and type `g_restart`.

If that doesn't work, try re-enabling that line, but replace `Z_BITS` with `32`.

**Comment by Raptor007** [comment]
Actually, I think I know why your ships are black.  It's probably the case-sensitive file system.

If you look at `Models/x-wing_hq.mtl` you'll see lines like this:
`map_Kd X-Wing_hq/TEX00009.bmp`

But the file is actually `Textures/X-Wing_hq/TEX00009.BMP` with a capitalized extension.

Try replacing `.bmp` with `.BMP` in all `.mtl` and `.obj` files.

**Comment by Raptor007** [comment]
I think the "Unable to set fullscreen video mode" issue was resolved in version 0.1.7 last year:

https://github.com/Raptor007/RaptorEngine/commit/4822227fb6712c1959774b2314a88f5e3bca4e1a#diff-4b22e1a7986f3e046aa3db3979d8cd34

But I haven't yet addressed `.bmp` vs `.BMP` on case-sensitive file systems.

**Comment by Raptor007** [comment]
[Version 0.1.9](https://www.mediafire.com/file/zsd8xeg4b7wwusf/X-Wing_Revival_0.1.9_Alpha.zip/file) should correct the case mismatches between models and textures.
